state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1947,7 +1944,7 @@ impl<T: Config<I>, I: 'static> DepositApi<T::TargetChain> for Pallet<T, I> {
 		source_asset: TargetChainAsset<T, I>,
 		destination_asset: Asset,
 		destination_address: ForeignChainAddress,
-		broker_commission_bps: BasisPoints,
+		broker_commission: Vec<Beneficiary<Self::AccountId>>,
```

Comment:
I understand that this is backwards compatible in the broker API, but is it possible to make the extrinsic backwards compatible? There are people building on the protocol that interact directly with these extrinsics.

Reply:
We were unsure about this and decided to just go with the breaking change. But I guess it can't hurt to avoid this.

We can keep (but deprecate) the old extrinsic and add a new one - this is the simplest way to keep compatibility. @marcellorigotti can you please change this?